packages/itwinui-react/src/core/Table/TableExpandableRowMemoized.tsx

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import * as React from 'react';
+import { Box, useMergedRefs } from '../../utils/index.js';
+import type { Virtualizer, VirtualItem } from '@tanstack/react-virtual';
+
+const TableExpandableRow = React.forwardRef(
+  (
+    props: {
+      virtualItem?: VirtualItem<Element>;
+      virtualizer?: Virtualizer<Element, Element>;
+      isDisabled: boolean;
+      children: React.ReactNode;
+    },
+    ref,
+  ) => {
+    const refs = useMergedRefs(ref, props.virtualizer?.measureElement);
+
+    return (
+      <Box
+        className={cx('iui-table-row', 'iui-table-expanded-content')}
+        style={{
```

Comment:
I have double-checked and it's needed since without this style content might shrink or grow out of the flex container when there isn't enough space. I have put it back to the style.